test/addFreakTest.js

```diff
@@ -0,0 +1,88 @@
+const { expect, assert } = require("chai");
+const EventEmitter = require('events');
+
+describe("FreakTeam", function () {
+  it("AddFreak fails when it's not called by hr", async function () {
+
+
+    const addFreak = await ethers.getContractFactory("FreakTeam");
```

Comment:
These two lines of code are duplicated on each test.
Please refactor the code to eliminate this duplication.